.ci/scripts/test.sh

```diff
@@ -213,7 +213,7 @@ elif [[ -n "${TAV_MODULE}" ]]; then
     memcached)
       DOCKER_COMPOSE_FILE=docker-compose-memcached.yml
       ;;
-    aws-sdk|@aws-sdk/client-s3)
+    aws-sdk|@aws-sdk/client-s3|@aws-sdk/client-sns)
```

Comment:
nit: This could perhaps be:
```suggestion
    aws-sdk|@aws-sdk/*)
```

I haven't tested that, however.